x-pack/plugins/security/public/management/role_mappings/role_mappings_grid/role_mappings_grid_page.tsx

```diff
@@ -129,71 +124,63 @@ export class RoleMappingsGridPage extends Component<Props, State> {
 
     if (loadState === 'finished' && roleMappings && roleMappings.length === 0) {
       return (
-        <EuiPageContent>
+        <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
           <EmptyPrompt history={this.props.history} />
         </EuiPageContent>
       );
     }
 
     return (
-      <EuiPageContent>
-        <EuiPageContentHeader>
-          <EuiPageContentHeaderSection>
-            <EuiTitle>
-              <h2>
-                <FormattedMessage
-                  id="xpack.security.management.roleMappings.roleMappingTitle"
-                  defaultMessage="Role Mappings"
-                />
-              </h2>
-            </EuiTitle>
-            <EuiText color="subdued" size="s">
-              <p>
-                <FormattedMessage
-                  id="xpack.security.management.roleMappings.roleMappingDescription"
-                  defaultMessage="Role mappings define which roles are assigned to users from an external identity provider. {learnMoreLink}"
-                  values={{
-                    learnMoreLink: (
-                      <EuiLink
-                        href={this.props.docLinks.links.security.mappingRoles}
-                        external={true}
-                        target="_blank"
-                      >
-                        <FormattedMessage
-                          id="xpack.security.management.roleMappings.learnMoreLinkText"
-                          defaultMessage="Learn more."
-                        />
-                      </EuiLink>
-                    ),
-                  }}
-                />
-              </p>
-            </EuiText>
-          </EuiPageContentHeaderSection>
-          <EuiPageContentHeaderSection>
+      <>
+        <EuiPageHeader
+          bottomBorder
+          pageTitle={
+            <FormattedMessage
+              id="xpack.security.management.roleMappings.roleMappingTitle"
+              defaultMessage="Role Mappings"
+            />
+          }
+          description={
+            <FormattedMessage
+              id="xpack.security.management.roleMappings.roleMappingDescription"
+              defaultMessage="Role mappings define which roles are assigned to users from an external identity provider. {learnMoreLink}"
+              values={{
+                learnMoreLink: (
+                  <EuiLink href={this.props.docLinks.links.security.mappingRoles} external={true}>
```

Comment:
Thanks @myasonik & @legrego , Yeah I'd like to push that decision down to @legrego's conversion. This one I want to try to keep to the basic setup. Changing those links is a slippery slope since they're all over management not just this one.